docs/development/docguide.rst

```diff
@@ -11,6 +11,20 @@ keep to these as closely as possible, and we encourage affiliated packages to
 also adhere to these as they encourage useful documentation, a characteristic
 often lacking in professional astronomy software.
 
+Adding a Git commit
+===================
+
+When your changes only affect documentation (i.e., docstring or RST files)
+and do not include any code snippets that require doctest to run, you may
+add a ``[skip travis]`` in your commit message. For example::
+
+    git commit -m "Update documentation about this and that [skip travis]"
+
+When this commit is pushed out to your branch associated with a pull request,
+Travis CI will be skipped because it is not required. This is because the
```

Comment:
Does this mean `[skip travis]` in practice now has the same effect as `[docs only]` (because last I recalled was  the decorators were not really honoured by CircleCI)?
[CONTRIBUTING.md](https://github.com/astropy/astropy/blob/master/CONTRIBUTING.md#other-tips) still has both recommendations, but it is not really clear there when and why to use which alternative – would be good to synchronise this.

Reply:
Hmm... The behavior is slightly different. Now I am confused too. @bsipocz ?

https://github.com/astropy/ci-helpers/blob/bffa3632d469a1c5dbec4f7293b8964adcbffe39/travis/setup_conda.sh#L41-L47

Reply:
Well, it would if there were any configurations left in `.travis.yml` with doctest-type entries in `SETUP_CMD` or `MAIN_CMD`. But since there aren't, seems to me the only difference is that `[skip travis]` will now be caught before that script is even executed?

From the point of documentation, recommending `[docs only]` still seems clearer to me, since it makes it more obvious that it restricts the tests to docs, without having to bother remembering that the docs tests are all run on CircleCI, and none on Travis (which hypothetically might change in the future again).

Reply:
`[docs only]` is our home grown workaround when there is any docs build on travis. Since we don't do that any more, `[skip travis]` is preferred as it is understand at the travis request level, and none of the jobs will start up as opposed to `[docs only]` where the jobs are still sitting in the CI queue and start up but then terminated before the installs happen.

Reply:
if we ever hack around for a `[docs only]` to be understand by circleCI, then I would certainly prefer that approach, but I leave the pleasure of figuring out how to do that to someone else.

Reply:
OK, it wasn't clear to me if `[skip travis]` kept them from ever being queued, but that's certainly a good reason. And probably cannot hurt if contributors remain vaguely aware where there PRs are being sent to.

Reply:
So is there something else I need to add to the doc, or is this just waiting for copyedit? Please let me know. Thanks for the reviews!

Reply:
yes. originally `[skip travis]` was also our own workaround that meant the jobs to start up but terminated before install, but it was a big enough and widely required feature request that they added it to be natively supported.

Reply:
This looks good to me, too. Would make sense to switch the recommendation in `CONTRIBUTING.md` to `[skip travis]` for all three cases as well.

Reply:
Ops, my bad for merging too soon. Please see #10199 . Thanks!